Cubical/Categories/Displayed/Constructions/Slice.agda

```diff
@@ -52,18 +56,11 @@ module _ (C : Category ℓC ℓC') (Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ') where
     test = refl
 
   Δ/C : Functorᴰ Id Cᴰ _/C_
-  Δ/C = mk∫ᴰFunctorᴰ _ Id (Wk.intro Id Id)
-    (BPᴰ.intro _
-      (Reindex.intro' F (reindF' _ Eq.refl Eq.refl (Disp.Snd {Cᴰ = Cᴰ})))
-      Fᴰ')
-    where
-      F = (∫F {Cᴰ = Cᴰ} (Wk.intro Id Id))
-      G = funcComp (Δ C) (Disp.Fst {Cᴰ = Cᴰ})
-      Gᴰ : Functorᴰ G (Unitᴰ (∫C Cᴰ)) (Hom C)
-      Gᴰ = (ID {C = C} ∘Fᴰ Unitᴰ.intro (Disp.Fst {Cᴰ = Cᴰ}))
-
-      Fᴰ' : Functorᴰ F (Unitᴰ (∫C Cᴰ)) (Hom C)
-      Fᴰ' = reindF' _ Eq.refl Eq.refl Gᴰ
+  Δ/C = TotalCatᴰ.introF _ _ (Wk.introF Id Id)
```

Comment:
I'm guessing that there is some gain in definitional behavior that motivates redefining this displayed slice. However, I don't necessarily see why

Reply:
I don't really understand the question. This construction isn't really being changed at all in this PR, it's just being updated because its dependencies changed.

As for where it's used, it's in the definition of a Fibration.